### ISSUE TITLE
When pasting to trilium from Windows clipboard, CR/LF are converted to spaces

### ISSUE THREAD

**Issue by taa1** [zadam/trilium#4]
Source text example of multiple lines beginning with a dash:
```
- Lorem ipsum dolor sit amet, consectetur adipiscing elit.
- Donec imperdiet orci eu arcu tincidunt varius.
- Fusce semper mi at sem interdum posuere.
- Pellentesque viverra erat et erat fringilla, condimentum feugiat dolor scelerisque.
- Vestibulum faucibus eros a nulla bibendum, at dignissim augue placerat.
- Praesent congue sem tristique convallis tempus.
```
When pasted into trilium, instead of a bulleted list:
- Lorem ipsum dolor sit amet, consectetur adipiscing elit.
- Donec imperdiet orci eu arcu tincidunt varius.
- Fusce semper mi at sem interdum posuere.
- Pellentesque viverra erat et erat fringilla, condimentum feugiat dolor scelerisque.
- Vestibulum faucibus eros a nulla bibendum, at dignissim augue placerat.
- Praesent congue sem tristique convallis tempus.

it's converted to a single paragraph:

```
- Lorem ipsum dolor sit amet, consectetur adipiscing elit.  - Donec imperdiet orci eu arcu tincidunt varius.  - Fusce semper mi at sem interdum posuere.  - Pellentesque viverra erat et erat fringilla, condimentum feugiat dolor scelerisque.  - Vestibulum faucibus eros a nulla bibendum, at dignissim augue placerat.  - Praesent congue sem tristique convallis tempus.
```

**Comment by zadam** [comment]
Yep, I noticed this as well. This issue comes from CKEditor which is used as the editing component - see https://github.com/ckeditor/ckeditor5/issues/766

I hope they will be able to fix this soon.

**Comment by zadam** [comment]
fixed in master with upgrade to CKEditor 10.1.0 - newlines are now preserved

Automatic conversion to bulleted list is not implemented (and is out of scope of this issue).
